ci/pipelines/Build.jenkinsfile

```diff
@@ -18,7 +17,15 @@ pipeline
 
 	parameters
 	{
-		booleanParam( name: 'uploadTestingPkg',
+		string(	name: 'version',
+				description: 'Version to build (must match the name of the tag that will be checked out), leave blank for checkout of current branch',
+				defaultValue: '' )
```

Comment:
Why not with 0.0.0 which is the default version value that will be used when building?

Reply:
When I refactor a pipeline I typically try to leave the version default as I encountered it. This was just maintaining the same philosophy. But I agree, I'll change it to 0.0.0. It makes more sense as no tag 0.0.0 exists.